static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
```

Comment:
ideally we should use a sass colour function to 'correct' something from our pallette?